### PR TITLE
feat(skymp5-server): implement ChangeValues message type

### LINKED ISSUE #193: Server must update value in `changeForm` when receives changed attribute percentage from the client

### PR DESCRIPTION
**Pull request by Dinakrab**
closes #193

### CODE REVIEW

File: skymp5-server/cpp/server_guest_lib/ActionListener.cpp [hunk 1]
```diff
@@ -449,3 +449,19 @@ void ActionListener::OnCustomEvent(const RawMessageData& rawMsgData,
...
+{
+  MpActor* actor = partOne.serverState.ActorByUser(rawMsgData.userId);
+  if (!actor) {
+    throw std::runtime_error("Unable to change values without Actor attached");
+  }
+  MpChangeForm changeForm = actor->GetChangeForm();
+  changeForm.healthPercentage = healthPercentage;
+  changeForm.magickaPercentage = magickaPercentage;
+  changeForm.staminaPercentage = staminaPercentage;
+  actor->ApplyChangeForm(changeForm);
```
Comment (Pospelove) [inline skymp5-server/cpp/server_guest_lib/ActionListener.cpp:466]: It seems that it works, but `ApplyChangeForm` is too heavy to use there. It's used to apply all character data loaded from a database. As far as I remember it also doesn't save change form automatically. Consider adding a method to the `MpActor` class that would update actor value percentages. Example: https://github.com/skyrim-multiplayer/skymp/blob/51b3da52de3d1d2ba5ddc18895112538f48a2264/skymp5-server/cpp/server_guest_lib/MpActor.cpp#L28

File: unit/ChangeValuesTest.cpp [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+TEST_CASE("Player attribute percentages are changing correctly",
+          "[ChangeValues] ")
+{
+  PartOne& p = GetPartOne();
+
+  DoConnect(p, 0);
+  p.CreateActor(0xff000000, { 0, 0, 0 }, 0, 0x3c);
+  p.SetUserActor(0, 0xff000000);
+  auto& ac = p.worldState.GetFormAt<MpActor>(0xff000000);
+  // ac.RegisterProfileId(pospelov);
```
Comment (Pospelove) [inline unit/ChangeValuesTest.cpp:60]: remove commented line of code

**Review by Pospelove — CHANGES_REQUESTED** [review]

**Review by Pospelove — APPROVED** [review]

**Comment by Pospelove** [comment]
LGTM. 
TODO: merge once CI is finished

**Comment by Dinakrab** [comment]
@Pospelove

**Pull request merged**